src/bentoml/io.py

```diff
@@ -10,6 +10,7 @@
 from ._internal.io_descriptors.pandas import PandasDataFrame
 from ._internal.io_descriptors.pandas import PandasSeries
 from ._internal.io_descriptors.text import Text
+from ._internal.io_descriptors.text_stream import TextStream
```

Comment:
also let's expose this in `src/bentoml/_internal/io_descriptor/__init__.py` as well for correctness